app/services/MediaApi.scala

```diff
@@ -28,10 +28,15 @@ class MediaApi(val config: ApplicationConfiguration, val ws: WSClient) {
   }
 
   def extractExportsFromJsonResponse(response: JsValue, cropId: String) = {
-    val exports = (response \ "data" \ "exports").as[List[Export]]
-    exports.find(_.id == cropId) match {
-      case Some(export) => Some(export.assets.sortBy(asset => asset.size).head.secureUrl)
-      case _ => None
+
+    try {
+      val exports = (response \ "data" \ "exports").as[List[Export]]
+      exports.find(_.id == cropId) match {
+        case Some(export) => Some(export.assets.sortBy(asset => asset.size).head.secureUrl)
+        case _ => None
+      }
+    } catch {
+      case e: Exception => None
```

Comment:
Why add this try/catch?

Reply:
If the image has been deleted from the grid, extracting these exports will fail. We still want to render the corresponding collection in the fronts tool, just without an image instead of failing to do so completely.

Reply:
Ah! Would parsing `JsValue` into an `Option[List[Export]]` and mapping over that be more intention revealing? Alternatively, could we add a comment as to why `Exception` will occur?